lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -1042,10 +1042,15 @@ def _stop_opensearch(self, *, restart=False) -> None:
         self.status.set(WaitingStatus(ServiceIsStopping))
 
         if self.opensearch.is_node_up():
-            # TODO: we should probably NOT have any exclusion on restart
-            # https://chat.canonical.com/canonical/pl/bgndmrfxr7fbpgmwpdk3hin93c
-            # 1. Add current node to the voting + alloc exclusions
-            self.opensearch_exclusions.add_current()
+            nodes = self._get_nodes(True)
```

Comment:
we should catch the `OpenSearchHttpError` here

Reply:
done